### DISCUSSION TITLE
【讨论】有没有发帖指南啊？没有的话建议加一个~

### DISCUSSION THREAD

**Discussion by SupulooZhao** [share121/inter-knot#1663] [其他]
比如标题的格式、比如图片的尺寸和字号等等。初来绳网感觉氛围很好，很是自由，但是希望能在此基础上有一些简单好遵守的发帖规则，让大家点进来的时候界面更整齐~~

**Comment by inkluv** [comment]
**规则不太清楚，格式教程挺多**
#999 #1092 #1361 还有很多
